java/cuvs-java/pom.xml

```diff
@@ -108,10 +109,11 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.13.0</version>
+                <version>3.11.0</version>
```

Comment:
The version of the `maven-compiler-plugin` had to be dropped to 3.11, to avoid build errors where Java22 might be built ahead of Java21.